src/app/domainregister/domainregister.component.ts

```diff
@@ -829,6 +835,22 @@ export class DomainRegisterComponent implements AfterViewInit {
     }
   }
 
+  public check_quota() {
+    const req = this.http.get('/rest/v1/email_hosting/domains_quota');
+    req.pipe(timeout(180000))
+      .subscribe((result: any) => {
+        this.domain_quota_allowed = result.result.domain_quota_allowed;
+        this.domain_quota_used = result.result.domain_quota_used;
+        if ( this.domain_quota_used >= this.domain_quota_allowed ) {
+            this.show_error('You have reached your domain quota allowed. Please purchase more Email Hosting products', 'Dismiss');
```

Comment:
Here as well:

`You have reached your allowed Email Domain quota. Please purchase more Email Hosting products.`